public/locales/en-US.json

```diff
@@ -372,8 +371,6 @@
     "label_billing_provider_azure": "Microsoft Azure",
     "label_billing_provider_gcp": "Google Cloud Platform",
     "label_billing_provider_oracle": "Oracle Cloud Infrastructure",
-    "label_billing_provider_red hat": "Red Hat Marketplace",
-    "label_category_cloud": "Public cloud",
```

Comment:
question: are we sure we're supposed to remove public cloud from the category listing? the `category` labels affect a dropdown listing associated with RHEL displays